website/docs/artist_inbox.md

```diff
@@ -20,14 +20,10 @@ Think of the inbox as a dynamic to-do list for tasks that require your attention
 The inbox has full keyboard support! Hover over any message and press `X` to mark it as read/unread or `C` to clear/unclear.
 :::
 
-### Important VS Other
-
-Currently, what qualifies as "Important" or "Other" is pre-set. However, future updates will allow you to customize the importance of messages.
-
 **Important**
 
 -   You are directly mentioned in the comment. For example: `Looks great @Felix Peterson`.
--   You are assigned to or removed from a task.
+-   You are a watcher on a task.
```

Comment:
I understand this text changes. However I wonder if it makes sense to also keep the old sentence in here, like `for example when you are assigned to or removed from a task` OR we add a bit that says something like "see the Importance Table about when you become a watcher." which may link to that title on this page directly?